build.gradle

```diff
@@ -14,28 +15,26 @@ gradlePlugin {
   plugins {
     updateChangelog {
       id = "${project.group}.updatechangelog"
-      description = 'A plugin that automatically updates the version in your changelog file.'
       implementationClass = "${project.group}.updatechangelog.UpdateChangelogPlugin"
     }
   }
 }
 
 repositories {
-  jcenter()
+  mavenCentral()
 }
 
 dependencies {
-  compile gradleApi()
-  compile localGroovy()
-  testCompile('org.junit.jupiter:junit-jupiter-api:5.3.1')
-  testRuntime('org.junit.jupiter:junit-jupiter-engine:5.3.1')
+  implementation gradleApi()
```

Comment:
this will be added by java-gradle-plugin, we don't need to do this ourselves: https://docs.gradle.org/current/userguide/java_gradle_plugin.html